api/src/dto/ChangeReqAckDto.ts

```diff
@@ -0,0 +1,16 @@
+import { DocType, Uuid, AckStatus } from "../types";
+import { IsNotEmpty, IsString } from "class-validator";
+
+/**
+ * API acknowledgement to client after client submission of an ChangeReqDto document.
+ */
+export class ChangeReqAckDto {
+    @IsNotEmpty()
+    docId: Uuid; // Uuid of submitted ChangeReqDto to be acknowledged
+    @IsNotEmpty()
+    type: DocType.ChangeReqAck;
+    @IsNotEmpty()
+    ack: AckStatus;
```

Comment:
Does this validate that it is one of the allowed enum values?

Reply:
Added an @Equals decorator to test for a specific value